evasquez26/angular4-startup
# Course teacher invitation

## Story / Task

see [ONE-57](https://bixlabs.atlassian.net/browse/ONE-57)


## What this PR does?

Invite teacher to a course.


### How to test / reproduce

- Enter a course where you are an admin.
- Click in the right panel the button 'Invite teacher'
- Search for a teacher.
- Click the plus sign.
- An invitation ticket should be created.

### TODOs



## Migrations
N/A


----
Thanks
